docs/CHANGELOG.asciidoc

```diff
@@ -8,16 +8,1981 @@
+
+[discrete]
+[[deprecation-136422]]
+.Deprecates ApiKey authentication for interactive users
+[%collapsible]
+====
+*Details* +
+The ability to authenticate interactive users authenticating with ApiKey has been deprecated, and will be removed in a future version. For more information, refer to {kibana-pull}136422[#136422].
+
+*Impact* +
+To authenticate interactive users, use <<kibana-authentication,another authentication method>>. 
+====
+      
+[float]
+[[features-8.4.0]]
+=== Features
+{kib} 8.4.0 adds the following new and notable features.
+
+Alerting::
+* Adds the "updated at" feature in new alerts table {kibana-pull}136949[#136949]
+* Adds a rule detail table with bulk actions {kibana-pull}136601[#136601]
+* Adds bulk Actions for Alerts Table {kibana-pull}135797[#135797]
+* Adds the Alerting stack-monitoring PoC {kibana-pull}135365[#135365]
+* Adds custom inline/row actions for alerts table {kibana-pull}134015[#134015]
+
+Discover::
+Adds the ability to add a custom number of rows in the results and save the specified number with a Saved Search {kibana-pull}135726[#135726]
+
+Elastic Security::
+For the Elastic Security 8.4.0 release information, refer to {security-guide}/release-notes.html[_Elastic Security Solution Release Notes_].
+
+Fleet::
+* Enables package signature verification feature {kibana-pull}137239[#137239]
+* Modal to allow user to force install an unverified package {kibana-pull}136108[#136108]
+* Display package verification status {kibana-pull}135928[#135928]
+* Tag rename and delete feature {kibana-pull}135712[#135712]
+* Bulk update agent tags ui {kibana-pull}135646[#135646]
+* Adds API to bulk update tags {kibana-pull}135520[#135520]
+* Adds and remove agent tags {kibana-pull}135320[#135320]
+* Support sorting agent list {kibana-pull}135218[#135218]
+* Promote Logstash output support to GA {kibana-pull}135028[#135028]
+* Create new API to manage download_source setting {kibana-pull}134889[#134889]
+
+Lens & Visualizations::
+* Adds supports for include and exclude terms in *Lens* {kibana-pull}136179[#136179]
+
+Machine Learning::
+* Adds random sampler to Data visualizer document count chart {kibana-pull}136150[#136150]
+* Adds explain log rate spikes feature to the ML plugin {kibana-pull}135948[#135948]
+
+Management::
+* Run packs live {kibana-pull}132198[#132198]
+* Ability to set human readable title of data view & ability to edit data view {kibana-pull}124191[#124191]
+
+Monitoring::
+Adds stale status reporting for Kibana {kibana-pull}132613[#132613]
+
+Observability::
+* Adds Beta label to Infrastructure tab {kibana-pull}136710[#136710]
+* Create and add Rule Alerts Summary as a sharable component to the O11y Rule Details {kibana-pull}135805[#135805]
+* Rule Details Page - Use RuleStatusPanel from triggersActionsUI {kibana-pull}135643[#135643]
+* Cases permissions can be customized to prevent a user from deleting Cases entities (cases themselves, and attachments/comments) {kibana-pull}135487[#135487]
```

Comment:
This feature applies to Cases in Observability, Security, and Stack Management, so should be moved into a separate `Cases` section.